from swype.exceptions.swype_error import SwypeError


class UnexpectedError(SwypeError):
    """ Raised for unknown or unexpected errors. """
    pass
